app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -292,12 +293,26 @@ public function updateOrganization(string $id, UpdateOrganizationRequest $reques
         );
     }
 
+    public function updateOrganizers(string $integrationId, UpdateIntegrationOrganizersRequest $request): RedirectResponse
+    {
+        $integration = $this->integrationRepository->getById(Uuid::fromString($integrationId));
+
+        $organizerIds = collect($integration->organizers())->map(fn (Organizer $organizer) => $organizer->organizerId);
+        $newOrganizers = array_filter(OrganizerMapper::map($request, $integrationId), static function (Organizer $organizer) use ($organizerIds) {
+            return !in_array($organizer->organizerId, $organizerIds->toArray(), true);
+        });
```

Comment:
```suggestion
        $newOrganizers = array_filter(
          OrganizerMapper::map($request, $integrationId),
          fn (Organizer $organizer) => !in_array($organizer->organizerId, $organizerIds->toArray(), true)
        );
```

This removes a bit of syntax overhead